licenses.json

```diff
@@ -9491,7 +9163,7 @@
     "publisher": "Web SDK Customer Support",
     "email": "web-sdk@onfido.com",
     "url": "https://github.com/onfido",
-    "path": "onfido-sdk-ui",
+    "path": "/home/ethanify/Work/onfido-sdk-ui",
```

Comment:
We should just keep this as "onfido-sdk-ui" from my discussion with @stefaniacardenas when reviewing this licenses.json file generator's implementation PR.